src/Installers/Windows/SharedFrameworkLib/SharedFrameworkLib.wixproj

```diff
@@ -40,4 +40,23 @@
   <PropertyGroup>
     <PackageFileName>$(InternalInstallerBaseName)-$(PackageVersion)-win-$(Platform)$(TargetExt)</PackageFileName>
   </PropertyGroup>
+
+  <!-- Overwrite the wix package drop creation target to create a lit
+       package instead. -->
+  <Target Name="CreateWixPackageDrop" AfterTargets="Build">
```

Comment:
Doesn't this project import src/Installers/Windows/Wix.targets through https://github.com/dotnet/aspnetcore/blob/release/5.0/src/Installers/Windows/Directory.Build.targets#L3❔ If yes, why do we need an override here❔

Reply:
This is a different form of package drop (Lit lib creator vs. Light compiler). Most of the projects create installers, while this project creates a wixlib